src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoHealthDataRecord.java

```diff
@@ -320,15 +320,15 @@ public void setUserDataGroups(Set<String> userDataGroups) {
     
     /** {@inheritDoc} */
     @Override
-    public Map<String, String> getUserSubstudyMemberships() {
-        return userSubstudyMemberships;
+    public Map<String, String> getUserStudyMemberships() {
+        return userStudyMemberships;
```

Comment:
The Exporter still reads this field by this name.

Reply:
Added a @DynamoDBAttribute to keep the existing column name... but is there a dependency on this Java code, or is it through the table alone?

Reply:
The dependency is through the DynamoDB tables directly.